lib/reporter.js

```diff
@@ -103,21 +102,23 @@ class SpecReporter extends events.EventEmitter {
         return symbol
     }
 
-    getColor (state) {
+    getColor (state, coloredLogs) {
```

Comment:
no need to pass in `coloredLogs` since we can just access it by `this.coloredLogs`. So just do:

```js
if (!this.coloredLogs) {
   return null
}
```